server/model/monitor.js

```diff
@@ -233,7 +233,15 @@ class Monitor extends BeanModel {
      * @returns {string}
      */
     encodeBase64(user, pass) {
-        return Buffer.from(user + ":" + pass).toString("base64");
```

Comment:
what is the type of `user`/`pass` in this context?
If it is an empty string, this should work as is, otherwise fixing the producer or adding   `|| ''` might be a better solution.

Reply:
> what is the type of user/pass in this context?
String